afs/templates/views/components/reports/scenes/name.htm

```diff
@@ -146,7 +145,15 @@ <h4 class="afs-block-key">Type: </h4>
                         </div>
                         <div class="afs-block-attributes">
                             <h4 class="afs-block-key">Exact Match: </h4>
-                            <div class="afs-block-value"><a data-bind="if: exactMatch() != '--', attr: {href: exactMatch}"><span data-bind="text: exactMatch"></span></a><span data-bind="if: exactMatch() == '--'">--</span></div>
+                            <div class="afs-block-value">
+                                <!-- ko foreach: exactMatch -->
+                                <a data-bind="if: $data != '--', attr: {href: $data}"><span data-bind="text: $data"></span></a>
+                                <span>;</span>
```

Comment:
Something like <!-- ko if: $index() > 0 -->,<!-- /ko --> might be better suited here so there's no trailing semicolon, or just toss that if onto the span itself.

Reply:
updated